### PR TITLE
Save all logos as Plain SVG

### PR DESCRIPTION
**Pull request by SebastJava**
* Xed edit all files (DELETE <title> elements causing new metadata to be generated)
* Save all files as Plain SVG (making files smaller, stripped from useless information)
* README.md "Save as Plain SVG and use pixels (px) as units"

### CODE REVIEW

**Comment by SebastJava** [comment]
@clefebvre  I don't want to put pressure. Take your time. But i have to explain something. I am creating a handful of other branches and PRs based on this branch here. So all my other PRs are looking awfully big. They are not really big, in fact. I was just assuming you would approve this PR here. I looked at all this deleted data here, and i don't find anything important in there. Except for this little extra info i added to the README.md...

**Short version:** As long as this #5 is neither approved or rejected, neither merged or closed, most of my other branches and PRs will feel awfully big, but they are not. They just depend on this, here.
